From: Miguel Telleria de Esteban <miguel@mtelleria.com>
Date: Wed, 23 May 2012 11:42:04 +0200
Subject: Defining static version in install to "3.03"

---
 install |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/install b/install
index ca6a864..1098b8c 100755
--- a/install
+++ b/install
@@ -61,7 +61,7 @@ for my $d ($ln, $to) {
 }
 
 chdir($ENV{GL_BINDIR});
-my $version = `git describe --tags --long --dirty=-dt`;
+my $version = "3.03";
 
 if ($to) {
     _mkdir($to);
